boards/px4/sitl/default.cmake

```diff
@@ -34,6 +34,7 @@ px4_add_board(
 		events
 		fw_att_control
 		fw_pos_control_l1
+		internal_resistance
```

Comment:
Can we name this more descriptively? Something that includes battery estimation (batt + est) perhaps?

Reply:
I changed it to battery_resistance_est. Let me know if you suggestion another.